helpers/integration.go

```diff
@@ -22,27 +22,31 @@ import (
 )
 
 const (
-	//HACBSTestOutputName is the name of the standardized HACBS Test output Tekton task result
-	HACBSTestOutputName = "HACBS_TEST_OUTPUT"
```

Comment:
We'll need to keep this constant so we can run in compatibility mode at least until after service preview.

Reply:
Never mind, totally missed the legacy name. Will have to put on my reading glasses :)